pybossa/forms/validator.py

```diff
@@ -37,7 +37,10 @@ def __init__(self, query_function, field_name, message=None):
         self.message = message
 
     def __call__(self, form, form_field):
-        filters = {self.field_name: form_field.data}
+        field_value = form_field.data
+        if isinstance(form_field, basestring):
+            field_value = form_field.data.strip()
```

Comment:
Should a unit test be included to cover this new line for code [coverage](https://coveralls.io/builds/28381697/source?filename=pybossa%2Fforms%2Fvalidator.py#L42)?

Reply:
Got it.